src/utils/daos_hdlr.h

```diff
@@ -102,6 +102,7 @@ struct cmd_args_s {
 	bool			verbose;	/* --verbose mode */
 	char			*entry;		/* --entry for ACL */
 	char			*principal;	/* --principal for ACL */
+	char			*name;		/* --name */
```

Comment:
Again, some field name reflecting the fact it is a property could be better, like for attr ?

Reply:
I was thinking in the struct it might be better to steer toward more generic names, since each command uses only a small subset of these values.

Reply:
You mean this field is shared by multiple sub-commands and not only for properties ?

Reply:
Not yet, but in theory it could.